package NQueryWizardFeatures;

message TWizardData {
    optional double CountPerRequest = 1;
    optional double Surplus = 2;
};

message TFeatures {
    optional double UsersPerDay = 1;
    optional TWizardData Images = 2;
    optional TWizardData Video = 3;
    optional TWizardData Weather = 4;
    optional TWizardData Music = 5;
    optional TWizardData Entity = 6;
    optional TWizardData Companies = 7;
    optional TWizardData Maps = 8;
    optional TWizardData Bno = 9;
    optional TWizardData Market = 10;
}

